Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -36,6 +37,8 @@ public class ModelOverrider : MonoBehaviour
 
         string m_BehaviorNameOverrideDirectory;
 
+        string m_OverrideExtension = ".nn";
```

Comment:
I think the extension should be `nn` not `.nn`. This should make it simpler to pass the extension as command line argument

Reply:
Updated.